src/main/java/uk/gov/hmcts/darts/common/repository/ObjectAdminActionRepository.java

```diff
@@ -29,4 +31,10 @@ public interface ObjectAdminActionRepository extends JpaRepository<ObjectAdminAc
     Optional<ObjectAdminActionEntity> findByTranscriptionDocument_IdAndObjectHiddenReasonIsNotNullAndObjectHiddenReason_MarkedForDeletionTrue(
         Integer transcriptionDocumentId);
 
+    @Query("""
+        SELECT o FROM ObjectAdminActionEntity o
+                WHERE o.markedForManualDelDateTime < :deletionThreshold
+        """)
+    List<ObjectAdminActionEntity> findFilesForManualDeletion(@Param("deletionThreshold") OffsetDateTime deletionThreshold);
```

Comment:
is `@Param("deletionThreshold")` annotation required?

Reply:
removed